### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by padrewin** [Cold-Development/MoneyPouchDeluxe#1] [Ideas]
Please share any suggestion you have and what you would like to see in a future update! It is more than welcomed. 😸

**Comment by Appliedtrout** [comment]
Hey, can you please allow more color code formats such as &#rrgbb&l which is the main color code format on bird flop and used by most servers?

Also can you please allow color codes in the Pouch lore's because they don't seem to work with even the most basic &f colors etc.

If u want feel free to contact me on discord. appliedtrout551
